README.md

```diff
@@ -231,16 +231,16 @@ and with a smooth evolutionary path.
 ## Memory safety
 
 Safety, and especially
-[memory safety](https://en.wikipedia.org/wiki/Memory_safety), remain key
-challenges for C++ and something a successor language needs to address. Our
-initial priority and focus is on immediately addressing important, low-hanging
+[memory safety](https://en.wikipedia.org/wiki/Memory_safety), remains a key
+challenge for C++ and something a successor language needs to address. Our
+initial priority and focus are on immediately addressing important, low-hanging
 fruit in the safety space:
 
 -   Tracking uninitialized states better, increased enforcement of
     initialization, and systematically providing hardening against
     initialization bugs when desired.
 -   Designing fundamental APIs and idioms to support dynamic bounds checks in
-    debug and hardened builds.
+    debugging and hardened builds.
```

Comment:
"debug builds" are a term of art, so should remain as such

Reply:
sure.